packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -61,19 +61,18 @@ export function decodeAttachment(attachment: media.MediaChunk, mimeType: string)
 
 /**
  * @hidden
- * Function returns null if the media call is supported on current mobile version, else SdkError.
- *
+ * Function throws an SdkError if the media call is not supported on current mobile version, else undefined.
+ * @throws an SdkError if the media call is not supported
  * @internal
  */
-export function isMediaCallSupportedOnMobile(mediaInputs: media.MediaInputs): SdkError {
+export function throwExceptionIfMediaCallIsNotSupportedOnMobile(mediaInputs: media.MediaInputs): void {
   if (isMediaCallForVideoAndImageInputs(mediaInputs)) {
-    return isApiSupportedOnMobile(videoAndImageMediaAPISupportVersion);
+    return throwExceptionIfMobileApiIsNotSupported(videoAndImageMediaAPISupportVersion);
```

Comment:
These returns are unnecessary now that throwExceptionIfMobileApiIsNotSupported returns void, can just call that function.

(I don't think the return statements are hurting anything, just unnecessary.)

Reply:
they are unnecessary and you're right they aren't hurting anything but to avoid confusion i will remove them